### DISCUSSION TITLE
The usage of "Place"-city locations

### DISCUSSION THREAD

**Discussion by haugekugga14** [pysal/momepy#672] [General]
Hey everyone, I’m new to momepy and I’m struggling with the correct way to specify city locations while using OSMnx. For example, I followed the tutorial and used:

tags = {"building": True}  
gdf = ox.features_from_place("Kahla, Germany", tags=tags)
gdf_projected = ox.projection.project_gdf(gdf)

This works perfectly, and I can plot the buildings successfully.

But when I change the location to something like:
gdf = ox.features_from_place("Rotterdam, Netherlands", tags=tags)

It throws an error. 😔

Furthermore, I’m trying to extract data for Shatin, one of the districts in Hong Kong. I’m unsure if I should write it as "Shatin, Hong Kong" or "Sha Tin District, Hong Kong" (or maybe something else entirely). I just don’t know the correct syntax for locations, and I keep getting errors.

**Comment by jGaboardi** [comment]
This does not relate to `momepy`, but usage of `osmnx`. Since this seems to be a usage issue of `osmnx` (not an actual bug) I suggest searching for a solution on Stack Overflow.

cc @gboeing
